pkgs/development/libraries/qt-5/5.15/default.nix

```diff
@@ -63,7 +63,18 @@ let
     ];
     qtscript = [ ./qtscript.patch ];
     qtserialport = [ ./qtserialport.patch ];
-    qtwebengine = lib.optionals stdenv.isDarwin [
+    qtwebengine = [
+      (fetchpatch {
+        url = "https://github.com/archlinux/svntogit-packages/raw/372ae3de526f783bdcb5d51b997fbf511055466a/trunk/qt5-webengine-python3.patch";
+        hash = "sha256-rUSDwTucXVP3Obdck7LRTeKZ+JYQSNhQ7+W31uHZ9yM=";
+      })
+      (fetchpatch {
+        url = "https://github.com/archlinux/svntogit-packages/raw/372ae3de526f783bdcb5d51b997fbf511055466a/trunk/qt5-webengine-chromium-python3.patch";
```

Comment:
I had no idea they've had added python 3 support so late, we should thank the Arch maintainers.

Reply:
It is not proper python3 support. eg one of the checks is just replaced with return true.
Is there some upstream thing we can link? That should hopefully make it easier to keep this in the future.

Reply:
Nothing upstream, the check is for the presence of a working python3 which we know we provide, so I think it's fine.